### DISCUSSION TITLE
Mail Scanner - unclear flow of information exchange

### DISCUSSION THREAD

**Discussion by GarethHumbel** [YetiForceCompany/YetiForceCRM#16493] [Feature improvement]
In the yetiforce mailscanner configuration here is impossible to use ticketing system in the way we all know from competitive systems. Example:

1. Customer write email
2.  User open created ticket and entering comment which is sent as a response to Customer email
3. Customer read above response and replaying accordingly
4. Yetiforce instead of appending client response to the list of comments of initial ticket, creating new ticket!

Above triggering mess which is unacceptable in even the most basic ticketing system.
Created comment doesnt contains any unique value (such as ticket number) which in normal world should be included and recognized by ticketing system, so certain email is tied to initially created ticket.
Mail Scanner (as the name indicating) should scan incoming emails and under specific designed conditions undertake certain action

This fundamental function has been forgotten or omitted intentionally. 
Are there any plans to fix it, so ticketing system can actually be used in yetiforce?

Respectfully, Gareth H

**Comment by s2b-git** [comment]
Maybe problem is with formatting subject line of responses?
